assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -30,41 +46,143 @@ const Title = styled.h3`
 	font-size: 19px;
 	line-height: 20px;
 	line-height: 1.4em;
-	flex: 2 0 0;
+	flex: 3 0 0;
 `;
 
 const Header = styled.div`
 	display: flex;
-	margin: 0px 4px 10px;
+	margin: 0px 0px 25px;
 `;
 
 const Message = styled.div`
-	 color: ${ ( { theme } ) => theme.colors.fg.v1 };
+	color: ${ ( { theme } ) => theme.colors.fg.v1 };
 	font-size: 19px;
 `;
 
+const FilterButtons = styled.div`
+	border-bottom:  2px solid ${ ( { theme } ) => theme.colors.mg.v1 };
+	padding: 18px 0px;
+	margin: 10px 0px 15px;
+`;
+
+const FilterButton = styled.button`
+	border: 0px;
+	background: none;
+	padding: 0px;
+	margin: 0px 28px 0px 0px;
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	font-size: 13px;
+	${ ( { active } ) => active && ` color: #fff; font-weight: bold; ` };
+`;
+
+const Search = styled.input`
+	width: 100%;
+	background: ${ ( { theme } ) => theme.colors.mg.v1 } !important;
+	border-color: ${ ( { theme } ) => theme.colors.mg.v1 } !important;
+	color: ${ ( { theme } ) => theme.colors.mg.v2 } !important;
+	padding: 2px 10px 2px 33px !important;
+	&::placeholder {
+	  color: ${ ( { theme } ) => theme.colors.mg.v2 };
+	}
+`;
+
+const SUPPORTED_IMAGE_TYPES = [
+	'image/png',
+	'image/jpeg',
+	'image/jpg',
+	'image/gif',
+];
+
+const SUPPORTED_VIDEO_TYPES = [
+	'video/mp4',
+];
+
+const FILTERS = [
+	{ filter: '', name: 'All' },
+	{ filter: 'image', name: 'Images' },
+	{ filter: 'video', name: 'Video' },
+];
+
+const DEFAULT_WIDTH = 150;
+
 function MediaLibrary( { onInsert } ) {
 	const {
-		state: { media, isMediaLoading, isMediaLoaded, mediaType },
-		actions: { loadMedia, setIsMediaLoading, setIsMediaLoaded },
+		state: { media, isMediaLoading, isMediaLoaded, mediaType, searchTerm },
+		actions: { loadMedia, setIsMediaLoading, setIsMediaLoaded, setMediaType, setSearchTerm },
 	} = useLibrary();
 
 	useEffect( loadMedia );
 
+	const isEven = ( n ) => {
+		return n % 2 === 0;
+	};
+
+	const getRelativeHeight = ( oWidth, oHeight, width ) => {
+		const racio = oWidth / width;
```

Comment:
Typo: should be `ratio`.

Reply:
🤦‍♂

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3907/commits/d02b0b72525a12bbc64d5ee2412ec3ea958a0665